### PR TITLE
Add log for connected peers

### PR DESCRIPTION
**Pull request by gaoqi8649**

### CODE REVIEW

File: src/network/taraxa_capability.cpp [hunk 1]
```diff
@@ -1110,16 +1105,9 @@ void TaraxaCapability::sendPbftVote(NodeID const &_id, taraxa::Vote const &vote)
 }
 
 void TaraxaCapability::onNewPbftBlock(taraxa::PbftBlock const &pbft_block) {
-  std::vector<NodeID> peers_to_send;
+  std::vector<NodeID> peers_to_send = getAllPeers();
   auto my_chain_size = pbft_chain_->getPbftChainSize();
-  {
-    boost::shared_lock<boost::shared_mutex> lock(peers_mutex_);
-    for (auto const &peer : peers_) {
-      if (!peer.second->isPbftBlockKnown(pbft_block.getBlockHash())) {
```
Comment (02p01r) [inline src/network/taraxa_capability.cpp:1118]: I guess you know what you're doing discarding this condition

**Review by 02p01r — APPROVED** [review]
LGTM

File: src/network/taraxa_capability.cpp [hunk 2]
```diff
@@ -1090,15 +1092,8 @@ void TaraxaCapability::logPacketsStats() {
 }
 
 void TaraxaCapability::onNewPbftVote(taraxa::Vote const &vote) {
-  std::vector<NodeID> peers_to_send;
-  {
-    boost::shared_lock<boost::shared_mutex> lock(peers_mutex_);
-    for (auto const &peer : peers_) {
-      if (!peer.second->isVoteKnown(vote.getHash())) {
```
Comment (02p01r) [inline src/network/taraxa_capability.cpp:1097]: I guess you know what you're doing discarding this condition

**Review by 02p01r — APPROVED** [review]

File: src/network/taraxa_capability.cpp (hunk 1 above)
Comment (gaoqi8649) [reply src/network/taraxa_capability.cpp]: My mistake

File: src/network/taraxa_capability.cpp (hunk 2 above)
Comment (gaoqi8649) [reply src/network/taraxa_capability.cpp]: My mistake

**Review by justinsnapp — APPROVED** [review]

**Pull request merged**